homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,31 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    
+    int fibN = 0;
+    if(sequence > 1) {
+        int fibMinus2 = 0;
+        int fibMinus1 = 1;
+        for(int i = 2; i <= sequence; ++i) {
+            fibN = fibMinus1 + fibMinus2;
+            fibMinus2 = fibMinus1;
+            fibMinus1 = fibN;
+        }
+    }    
+    else {
+        return sequence;
+    }
+    return fibN;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+
+    if (sequence >= 3) {
+        return fibonacci_recursive(sequence - 1) + fibonacci_recursive(sequence - 2);
+    }
+    else if (sequence > 0) {
+        return 1;
+    }
```

Comment:
`else if` is redundant here.


If you return variable here, you don't need to use `else if`